Update README.md

ShannonHeylmun opened the pull request:
needs explain

ShannonHeylmun committed:
Update README.md

ShannonHeylmun committed:
Add some content

zheylmun commented on README.md:
```diff
@@ -1 +1,13 @@
+
+[Built with Zola](https://www.getzola.org)
+
+## Please steal this
```
This is a little weird to say, as I'm not offering anything beyond what the Apollo theme already offers.
It probably makes more sense to point people at how to do that.

ShannonHeylmun committed:
Address feedback and add link to Apollo theme

https://github.com/zheylmun/site/pull/2/commits/7a62a9e1b1874b1104d221a48786ef703585c899#r1947293814

ShannonHeylmun committed:
Rewording

ShannonHeylmun committed:
Merge branch 'zheylmun:main' into patch-1

ShannonHeylmun committed:
Shortened README.md

ShannonHeylmun committed:
reword

ShannonHeylmun commented:
Made a couple rewordings on the "about me" page, too.
